pyphare/pyphare/pharesee/tovtk.py

```
@@ -0,0 +1,240 @@
+        lvl = root.create_group(f"Level{ilvl}")
+        lvl_spacing = root_spacing
+        lvl_spacing = [dl / 2**ilvl for dl in lvl_spacing] + [0.0]
+        lvl.attrs.create("Spacing", lvl_spacing, dtype="f")
+        steps_lvl = steps.create_group(f"Level{ilvl}")
+
+        AMRBox = []
+        step_nbrBoxes = []
+        AMRBoxOffsets = []
+        dataOffsets = []
+
+        cellData_g = lvl.create_group("CellData")
```

Ruff (0.8.2)

Local variable `cellData_g` is assigned to but never used

Remove assignment to unused variable `cellData_g`

(F841)